client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -156,6 +167,18 @@ export const useStoreItemInfo = ( {
 		[ getPurchase, isOwned, isSuperseded, isUpgradeableToYearly, sitePlan, translate ]
 	);
 
+	const isMultisiteCompatible = useCallback( ( item: SelectorProduct ) => {
+		if ( isJetpackPlanSlug( item.productSlug ) ) {
+			// plans containing Jetpack Backup and/or Jetpack Scan are incompatible with multisite installs
+			return ! [ ...JETPACK_BACKUP_PRODUCTS, ...JETPACK_SCAN_PRODUCTS ].filter( ( productSlug ) =>
+				planHasFeature( item.productSlug, productSlug )
+			).length;
+		}
+		return ! (
+			[ ...JETPACK_BACKUP_PRODUCTS, ...JETPACK_SCAN_PRODUCTS ] as ReadonlyArray< string >
+		 ).includes( item.productSlug );
+	}, [] );
```

Comment:
Since this has no dependencies, it can be moved outside the hook just like `isDeprecated` above there.